lib/3.7/services.cf

```diff
@@ -905,49 +905,49 @@ bundle agent classic_services(service,state)
                         canonify("$(inits)_set"));
 
   reports:
-    inform_mode::
-      "$(this.bundle): Using init system $(inits)"
+    DEBUG|DEBUG_classic_services::
+      "DEBUG $(this.bundle): Using init system $(inits)"
```

Comment:
When the variable class guard was in use here it was causing a programming error.